src/app/core/models/base-issue.model.ts

```diff
@@ -44,9 +46,16 @@ export class BaseIssue implements Issue {
     this.created_at = moment(githubIssue.created_at).format('lll');
     this.title = githubIssue.title;
     this.description = githubIssue.body;
+
+    this.severity = githubIssue.findLabel('severity');
```

Comment:
Should we also introduce the ability to check for multiple label types within an issue? For example `severity.High` & `severity.Low` are both assigned to an issue. (Especially in the case when the user is using Github directly instead of our app)

Reply:
Also should we use some sort of enum to restrict the inputs? Instead of passing in a direct string input? Perhaps one set of enums for CategoricalLabels and another for Non-CategoricalLabels. That way the `findLabel` method can simply take in 2 optional enums (either Categorical or NonCategorical) and can throw an error if both are input / none are input. Would that be a good idea? I'm not sure 😅

Reply:
> Should we also introduce the ability to check for multiple label types within an issue? For example `severity.High` & `severity.Low` are both assigned to an issue. (Especially in the case when the user is using Github directly instead of our app)
Sure, if the rank of labels in the category is specified, it will get the highest ranked label. If the ranks are not specified, the method will return the first label of the category it is able to find.

> Also should we use some sort of enum to restrict the inputs? Instead of passing in a direct string input? Perhaps one set of enums for CategoricalLabels and another for Non-CategoricalLabels. That way the `findLabel` method can simply take in 2 optional enums (either Categorical or NonCategorical) and can throw an error if both are input / none are input. Would that be a good idea? I'm not sure
Not a fan of having 2 optional parameters tho. It will lead to function calls like `example(undefined, variable)`.

Reply:
> Sure, if the rank of labels in the category is specified, it will get the highest ranked label. If the ranks are not specified, the method will return the first label of the category it is able to find.

Great !

> Not a fan of having 2 optional parameters tho. It will lead to function calls like `example(undefined, variable)`.

Ah right thats true. Yeah i guess we can leave it as it is. 👍